docs/advanced.md

```diff
@@ -0,0 +1,62 @@
+# Advanced Options
+
+{% hint style="warning" %}
+This is a section detailing the more advanced parts of using Northstar, with some of them having a chance to potentially mess with your game. Setting your level too high, for example, will cause you to need to reset your level back to 1.
+{% endhint %}
+
+## Mod profiles <a href="#profiles" id="profiles"></a>
+
+Profiles are a way to launch a version of Northstar with specific mods enabled, differing per profile. These profiles are seperated by folders that you create yourself, and add your own mods to. This is especially useful if you want to [play vanilla using Northstar](advanced.md#vanilla-on-northstar). When launching using profiles, you should make sure EA/Origin are open before launching as to make sure you encounter less issues.
+
```

Comment:
Might also wanna mention mod-managers here ^^

```suggestion

Note that VTOL and r2modman are the only mod-managers that currently support profiles.

```

Maybe even link to them (don't have the link on hand though)